<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calendar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeWidget" name="treeView">
        <column>
         <property name="text">
          <string>Arbre des projets</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <widget class="QWidget" name="horizontalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>301</width>
            <height>381</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="maximumSize">
                <size>
                 <width>297</width>
                 <height>186</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>18</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Bienvenue !</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QPushButton" name="pushButtonImport">
                 <property name="text">
                  <string>Importer un calendrier</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>291</width>
            <height>381</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Description du projet</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Nom :</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelNomProjet">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Disponibilité :</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelDispoProjet">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_11">
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Echéance :</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelEcheProjet">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <widget class="QWidget" name="horizontalLayoutWidget_9">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>301</width>
            <height>381</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <family>Calibri</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Description de la tâche</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_14">
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Identificateur :</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelIdTache">
                   <property name="minimumSize">
                    <size>
                     <width>130</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_15">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_16">
                   <item>
                    <widget class="QLabel" name="label_8">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Disponibilité :</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelDispoTache">
                   <property name="minimumSize">
                    <size>
                     <width>130</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_17">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_18">
                   <item>
                    <widget class="QLabel" name="label_9">
                     <property name="maximumSize">
                      <size>
                       <width>130</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Echéance :</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelEcheTache">
                   <property name="minimumSize">
                    <size>
                     <width>130</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_21">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_22">
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Type :</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="labelTypeTache">
                 <property name="minimumSize">
                  <size>
                   <width>130</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>130</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QStackedWidget" name="stackedWidget_2">
                 <property name="maximumSize">
                  <size>
                   <width>267</width>
                   <height>60</height>
                  </size>
                 </property>
                 <widget class="QWidget" name="page_4">
                  <widget class="QWidget" name="horizontalLayoutWidget_15">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>267</width>
                     <height>31</height>
                    </rect>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_23">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_24">
                      <item>
                       <widget class="QLabel" name="label_12">
                        <property name="maximumSize">
                         <size>
                          <width>16777215</width>
                          <height>20</height>
                         </size>
                        </property>
                        <property name="font">
                         <font>
                          <pointsize>11</pointsize>
                         </font>
                        </property>
                        <property name="text">
                         <string>Préemptive :</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QLabel" name="labelPreemptTache">
                      <property name="minimumSize">
                       <size>
                        <width>130</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>130</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="horizontalLayoutWidget_16">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>30</y>
                     <width>267</width>
                     <height>31</height>
                    </rect>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_25">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_26">
                      <item>
                       <widget class="QLabel" name="label_13">
                        <property name="font">
                         <font>
                          <pointsize>11</pointsize>
                         </font>
                        </property>
                        <property name="text">
                         <string>Durée :</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QLabel" name="labelDureeTache">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>130</horstretch>
                        <verstretch>20</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>130</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>130</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                 <widget class="QWidget" name="page_5"/>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_19">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetDefaultConstraint</enum>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Précédences :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_20">
                   <item>
                    <widget class="QListWidget" name="listPred">
                     <property name="maximumSize">
                      <size>
                       <width>208</width>
                       <height>100</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProjet">
    <property name="title">
     <string>Projet</string>
    </property>
    <addaction name="actionCreerNouvProj"/>
    <addaction name="actionSupprimet_Projet"/>
   </widget>
   <widget class="QMenu" name="menuTache">
    <property name="title">
     <string>Tâche</string>
    </property>
    <widget class="QMenu" name="menuPr_c_dence">
     <property name="title">
      <string>Précédence</string>
     </property>
     <addaction name="actionAjouter_Pr_decende"/>
     <addaction name="actionSupprimer_2"/>
    </widget>
    <widget class="QMenu" name="menuSous_T_che">
     <property name="title">
      <string>Sous-Tâche</string>
     </property>
     <addaction name="actionAjouter_Sous_T_che"/>
     <addaction name="actionSupprimer"/>
    </widget>
    <addaction name="actionNouvelleTache"/>
    <addaction name="actionSupprimer_T_che"/>
    <addaction name="menuPr_c_dence"/>
    <addaction name="menuSous_T_che"/>
   </widget>
   <widget class="QMenu" name="menuSemaine">
    <property name="title">
     <string>Emploi du temps</string>
    </property>
    <addaction name="actionAffichage_3"/>
   </widget>
   <widget class="QMenu" name="menuEvenement">
    <property name="title">
     <string>Programmation</string>
    </property>
    <widget class="QMenu" name="menuT_che">
     <property name="title">
      <string>Tâche</string>
     </property>
     <addaction name="actionProgrammation_d_une_t_che"/>
     <addaction name="actionD_programmer_T_che"/>
    </widget>
    <widget class="QMenu" name="menuActivit">
     <property name="title">
      <string>Activité</string>
     </property>
     <addaction name="actionProgAct"/>
     <addaction name="actionD_programmer_Activit"/>
    </widget>
    <addaction name="menuT_che"/>
    <addaction name="menuActivit"/>
   </widget>
   <widget class="QMenu" name="menuCalendrier">
    <property name="title">
     <string>Calendrier</string>
    </property>
    <addaction name="actionImporter"/>
    <addaction name="actionExporter_2"/>
   </widget>
   <addaction name="menuCalendrier"/>
   <addaction name="menuProjet"/>
   <addaction name="menuTache"/>
   <addaction name="menuEvenement"/>
   <addaction name="menuSemaine"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionCreerNouvProj">
   <property name="text">
    <string>Nouveau Projet</string>
   </property>
  </action>
  <action name="actionAffichage">
   <property name="text">
    <string>Affichage</string>
   </property>
  </action>
  <action name="actionNouvelleTache">
   <property name="text">
    <string>Nouvelle Tâche</string>
   </property>
  </action>
  <action name="actionAffichageTache">
   <property name="text">
    <string>Affichage</string>
   </property>
  </action>
  <action name="actionRechercheTache">
   <property name="text">
    <string>Recherche</string>
   </property>
  </action>
  <action name="actionAffichage_3">
   <property name="text">
    <string>Affichage</string>
   </property>
  </action>
  <action name="actionProgAct">
   <property name="text">
    <string>Programmer</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="actionExporter">
   <property name="text">
    <string>Exporter</string>
   </property>
  </action>
  <action name="actionAjouter_Pr_decende">
   <property name="text">
    <string>Ajout</string>
   </property>
  </action>
  <action name="actionProgrammation_d_une_t_che">
   <property name="text">
    <string>Programmer</string>
   </property>
  </action>
  <action name="actionAjouter_Sous_T_che">
   <property name="text">
    <string>Ajout</string>
   </property>
  </action>
  <action name="actionD_programmer_Activit">
   <property name="text">
    <string>Déprogrammer</string>
   </property>
  </action>
  <action name="actionD_programmer_T_che">
   <property name="text">
    <string>Déprogrammer</string>
   </property>
  </action>
  <action name="actionSupprimer_T_che">
   <property name="text">
    <string>Supprimer Tâche</string>
   </property>
  </action>
  <action name="actionSupprimet_Projet">
   <property name="text">
    <string>Supprimer Projet</string>
   </property>
  </action>
  <action name="actionImporter">
   <property name="text">
    <string>Importer</string>
   </property>
  </action>
  <action name="actionExporter_2">
   <property name="text">
    <string>Exporter</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionSupprimer_2">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
